config/initializers/scheduler.rb

```diff
@@ -0,0 +1,12 @@
+require 'rufus-scheduler'
+
+s = Rufus::Scheduler.singleton
```

Comment:
Ez a file minden processben lefut es mindenhol lesz egy schedulered es tobbszor fogod lefuttatni a metascoringot.

A rufus-scheduler helyett lehetne a sidekiq-cron gemet hasznalni: https://github.com/ondrejbartas/sidekiq-cron.
Belul az is rufus-schedulert hasznal viszont megold egy csomo problemat, amire ti esetleg nem gondoltok.